boxoffice/__init__.py

```diff
@@ -1,76 +1,37 @@
 # -*- coding: utf-8 -*-
 # imports in this file are order-sensitive
-import re
 from pytz import timezone
 from flask import Flask
-from flask.ext.assets import Environment, Bundle
+from flask.ext.mail import Mail
 from flask.ext.lastuser import Lastuser
 from flask.ext.lastuser.sqlalchemy import UserManager
-from baseframe import baseframe, baseframe_js, baseframe_css
 import coaster.app
+import wtforms_json
+
 
 app = Flask(__name__, instance_relative_config=True)
 lastuser = Lastuser()
+ALLOWED_ORIGINS = ['http://shreyas-wlan.dev:8000',
+                   'http://rootconf.vidya.dev:8090',
+                   'http://rootconf.karthik.dev:8090']
```

Comment:
Should this be here? It sounds like it should be in settings.py, or better, in the database.